.circleci/config.yml

```
@@ -157,6 +187,9 @@ jobs:
       - save-cache
 
 workflows:
+  detect-breaking-changes:
+    jobs:
+      - detect-breaking-changes
```

NIT: We use custom names in other jobs
```suggestion
      - detect-breaking-changes:
          name: "Detect Breaking Changes"
```